app/app.go

```diff
@@ -1019,28 +1019,51 @@ func (app *PstakeApp) RegisterUpgradeHandler() {
 	app.UpgradeKeeper.SetUpgradeHandler(
 		UpgradeName,
 		func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-			// stuck unbonding epoch numbers
-			RemovableUnbondings := map[string]map[int64]any{"cosmoshub-4": {312: nil}, "osmosis-1": {429: nil, 432: nil}}
-
-			// get the stuck unbondings from the store
-			unbondings := app.LiquidStakeIBCKeeper.FilterUnbondings(
-				ctx,
-				func(u liquidstakeibctypes.Unbonding) bool {
-					_, chain := RemovableUnbondings[u.ChainId]
-					if chain {
-						_, epoch := RemovableUnbondings[u.ChainId][u.EpochNumber]
-						if epoch {
-							return true
-						}
-					}
-					return false
-				},
-			)
-
-			// mark the stuck unbondings as failed, so they can be processed
-			for _, unbonding := range unbondings {
-				unbonding.State = liquidstakeibctypes.Unbonding_UNBONDING_FAILED
-				app.LiquidStakeIBCKeeper.SetUnbonding(ctx, unbonding)
+			for _, hc := range app.LiquidStakeIBCKeeper.GetAllHostChains(ctx) {
+				switch hc.ChainId {
+				case "cosmoshub-4":
+					upperLimit, _ := sdk.NewDecFromStr("1.01")
+					lowerLimit, _ := sdk.NewDecFromStr("0.85")
+					hc.Params.UpperCValueLimit = upperLimit
+					hc.Params.LowerCValueLimit = lowerLimit
+					app.LiquidStakeIBCKeeper.SetHostChain(ctx, hc)
+					break
```

Comment:
continue?

Reply:
it breaks the switch statement, not the for loop

Reply:
then its redundant.. check lint too